src/main.ts

```diff
@@ -122,10 +147,21 @@ export async function run(): Promise<void> {
     await rm(path)
 
     core.debug(`  - Removed: "${path}"`)
+
+    artifactRows.push([
+      relativePath,
+      humanSize,
+      `<a href="${publicUrl}">${publicUrl}</a>`
```

Comment:
I think we should run `relativePath` and `publicUrl` through something like [`escape-html`](https://www.npmjs.com/package/escape-html?activeTab=code). It is unlikely that someone would create paths with `"`, `&`, `'`, `<` or `>` in them by accident or that a bit of malformed HTML would produce any actual problems.
But I think one should always keep these kind of input validations / sanitization in mind, regardless of their assumed effect.

Reply:
The output is parsed as markdown, which only allows specific HTML markup. If we need this, markdown syntax would need to be escaped as well.
GitHub also parses the stdout for annotations, so we need to consider this kind of output to be under the control of any code running in the job.